### ISSUE TITLE
Black box and keys are not displayed on minimap when carried

### ISSUE THREAD

**Issue by rbcat** [colobot/colobot#1241]
When you have radar station installed and a grabber or a wasp takes BlackBox, KeyA, KeyB or etc, it disappears from minimap.

**Comment by tomangelo2** [comment]
Confirmed to be happening in original game as well.

**Comment by krzys-h** [comment]
Is it really a bug though? Seems by-design to me, you would have two minimap icons on top of each other so the one below could not be clicked in any way.

**Comment by rbcat** [comment]
There could be two exception cases, though - carrier is a wasp and there is no radar station or carrier is a bot which you cannot select (has unselectable=1 set).

**Comment by krzys-h** [comment]
Wouldn't that break the design of the crazy robot level though? You could just wait until it runs out of power since you can see the blackbox on the map. Unless there is no RadarStation in that level, which would make no sense since you already built one in previous missions.

**Comment by rbcat** [comment]
But, there isn't any radar station in that level and you cannot build anything, that is. Not only that, inability to use one seems like a purposely feature of these two levels, as for me.

**Comment by krzys-h** [comment]
Yeah, my bad, having a RadarStation there would break the level anyway
